### PR TITLE
SSCS-10574: Adding mandatory field "dwpIsOfficerAttending" into request

### PR DESCRIPTION
**Pull request by HarryH96**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/SSCS-10574

### Change description ###
- Adding mandatory field into request payload to fix tests

**Does this PR introduce a breaking change?** (check one with "x")

```
[x] Yes - Requires CCD PR: https://github.com/hmcts/sscs-ccd-definitions/pull/970 to be merged first
[] No
```

### CODE REVIEW

**Review by MohammedHMCTS — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/dwpuploadresponse/DwpUploadResponseAboutToSubmitHandler.java [hunk 1]
```diff
@@ -108,6 +108,10 @@ public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType,
             }
         }
 
+        if (YesNo.isNoOrNull(sscsCaseData.getDwpFurtherInfo())) {
```
Comment (dattipoeHMCTS) [inline src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/dwpuploadresponse/DwpUploadResponseAboutToSubmitHandler.java:111]: I don't think this change needs to be added - functionality already exists

File: src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/dwpuploadresponse/DwpUploadResponseAboutToSubmitHandlerTest.java [hunk 2]
```diff
@@ -1359,4 +1359,21 @@ public void givenNonSscs5CaseAndCaseCodeIsSetToSscs5Code_thenErrorIsShown(String
         assertThat(response.getWarnings().size(), is(0));
         assertEquals("Benefit code cannot be changed to the selected code", response.getErrors().stream().findFirst().get());
     }
+
+    @Test
+    public void givenADwpUploadResponseEventWithoutFurtherInformation_thenSetStateToReadyToList() {
```
Comment (dattipoeHMCTS) [inline src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/dwpuploadresponse/DwpUploadResponseAboutToSubmitHandlerTest.java:1364]: Same with this

File: dependency-check-suppressions.xml [hunk 3]
```diff
@@ -156,4 +156,31 @@
         <packageUrl regex="true">^pkg:maven/org.springframework/spring-web@.*$</packageUrl>
         <cve>CVE-2016-1000027</cve>
     </suppress>
+    <suppress>
```
Comment (dattipoeHMCTS) [inline dependency-check-suppressions.xml:159]: Haven't the suppressions been done?

**Review by dattipoeHMCTS — CHANGES_REQUESTED** [review]

**Comment by HarryH96** [comment]
Tests seem to now be passing - these changes dont seem to be required

**Pull request closed without merging**